src/hooks/useLocalStorage/useLocalStorage.ts

```diff
@@ -0,0 +1,25 @@
+import { useState } from "react";
+
+const useLocalStorage = (key: string, initial: boolean | string) => {
```

Comment:
w takich przypadkach gdzie masz wiele możliwych typów możesz stworzyć typ generyczny, czyli taki który podasz w momencie wykorzystania useLocalStorage i będzie on definiował jakiego typu jest  'initial'

Reply:
ale to takie 'nice to have' i całkiem złożona sprawa - jeśli się uda spoko, jeśli nie też spoko :D